src/features/user/Settings/EditProfile/AddressManagement/EditAddress.tsx

```diff
@@ -23,6 +23,11 @@ interface Props {
   showPrimaryAddressToggle: boolean;
 }
 
+type EditAddressApiPayload = FormData & {
+  country: ExtendedCountryCode | string;
+  type: AddressType;
+};
```

Comment:
Delink EditAddressApiPayload from FormData as FormData permits null/undefined values for mandatory types.

Reply:
I have updated the FormData type to no longer accept null or undefined for the mandatory fields.